### PR TITLE
[Kriso] Remove left- and right-wildcards

### PR DESCRIPTION
**Pull request by Bisaloo**
https://github.com/EFForg/https-everywhere/issues/11203

### CODE REVIEW

**Comment by Bisaloo** [comment]
Disabled because @cschanaj found it broken in #11200

File: src/chrome/content/rules/Krisostomus.xml [hunk 1]
```diff
@@ -1,30 +1,40 @@
...
+		http://www.kriso.ee/language-su-C-english2.html
+		http://www.kriso.ee/technology-engineering-agriculture-su-T-english2.html
 
-		- kriso.lt *
-		- kriso.lv *
+	Invalid certificate:
+		blog.kriso.ee
+		tarturatsakool.kriso.ee
+		test.kriso.ee
+		winserver.kriso.ee
```
Comment (J0WI) [inline src/chrome/content/rules/Krisostomus.xml]: `static.kriso.ee`

File: src/chrome/content/rules/Krisostomus.xml [hunk 2]
```diff
@@ -1,30 +1,40 @@
 <!--
-	Problematic domains:
+	Breaks search by subject on www.kriso.ee
+		http://www.kriso.ee/health-personal-development-su-V-english2.html
+		http://www.kriso.ee/language-su-C-english2.html
+		http://www.kriso.ee/technology-engineering-agriculture-su-T-english2.html
```
Comment (J0WI) [inline src/chrome/content/rules/Krisostomus.xml]: How can I reproduce this issue? Search seems to work fine for me.

Comment (Bisaloo) [inline src/chrome/content/rules/Krisostomus.xml]: I copied the comment from #11200 but maybe it's simpler to label it as `Different content http/https`.

**Pull request merged**